source/acquaman/BioXAS/BioXASMainXASScanConfiguration.h

```diff
@@ -50,6 +54,8 @@ protected slots:
 protected:
 	/// Method that does all the calculations for calculating the estimated scan time.
 	virtual void computeTotalTimeImplementation();
+    /// Holds the total time in seconds that the scan is estimated to take.
+    double totalTime_;
```

Comment:
Hey @dleacock! I think the totalTime_ may already be a member of BioXASScanConfiguration, an ancestor of both BioXASSideXASScanConfiguration and BioXASMainXASScanConfiguration. It's possible that you don't need to re-declare it here.